public/sw_global_variables.js

```diff
@@ -12,24 +12,24 @@ const cacheVersion = 'cache v1.3';
  * @PROD
  * Make sure the URL is the EXACT same as what the browser fetches. Letter casing is important.
  */
-const apiSource = String('https://360api.gordon.edu/').toLowerCase() + 'api';
+// const apiSource = String('https://360api.gordon.edu/').toLowerCase() + 'api';
 /**
  * @TRAIN
  * Make sure the URL is the EXACT same as what the browser fetches. Letter casing is important.
  */
-// const apiSource = String('https://360apitrain.gordon.edu/').toLowerCase() + 'api';
+const apiSource = String('https://360apitrain.gordon.edu/').toLowerCase() + 'api';
```

Comment:
I don't know, but probably not since we are not production. I don't really know what it does, but it seemed to fix a few errors. It would probably be best to ask an admin about it